platform-configs/include/protectli-vp24xx.robot

```diff
@@ -12,3 +12,5 @@ ${MAX_CPU_TEMP}=                95
 ${DMIDECODE_MANUFACTURER}=      Protectli
 ${DMIDECODE_VENDOR}=            3mdeb
 ${DMIDECODE_FAMILY}=            Vault Pro
+
+${ETHERNET_ID}=                 8086:1539
```

Comment:
I see vp2420 has different port than vp2410. VP2430 will also have different. I would move it to the board-specific config rather than include into common vp24xx config.

Reply:
k makes sense, https://github.com/Dasharo/open-source-firmware-validation/pull/638/commits/30d89c0f00cc0495d0fe4d91dbdbcb01bc62d2a3